Source/JavaScriptCore/inspector/remote/cocoa/RemoteInspectorCocoa.mm

```diff
@@ -72,15 +72,17 @@ static bool canAccessWebInspectorMachPort()
     return !sandbox_check(getpid(), "mach-lookup", static_cast<enum sandbox_filter_type>(SANDBOX_FILTER_GLOBAL_NAME | SANDBOX_CHECK_NO_REPORT), WIRXPCMachPortName);
 }
 
-static bool globalAutomaticInspectionState()
+void RemoteInspector::updateFromGlobalNotifyState()
 {
     int token = 0;
-    if (notify_register_check(WIRAutomaticInspectionEnabledState, &token) != NOTIFY_STATUS_OK)
-        return false;
+    if (notify_register_check(WIRGlobalNotifyStateName, &token) != NOTIFY_STATUS_OK)
+        return;
+
+    uint64_t state = 0;
+    notify_get_state(token, &state);
 
-    uint64_t automaticInspectionEnabled = 0;
-    notify_get_state(token, &automaticInspectionEnabled);
-    return automaticInspectionEnabled == 1;
+    m_automaticInspectionEnabled = state & WIRGlobalNotifyStateAutomaticInspection;
+    m_simulateCustomerInstall = state & WIRGlobalNotifyStateSimulateCustomerInstall;
```

Comment:
I think we should also put this behind `allowInternalSecurityPolicies` (and make sure not to even send it inside `webinspectord` using a similar check) so that we don't even bother with this logic when not dealing with a development system.